tests/acceptance/new-test.js

```diff
@@ -35,11 +36,11 @@ describe('Acceptance: ember new', function () {
     } else {
       process.env.CI = ORIGINAL_PROCESS_ENV_CI;
     }
-    return tmp.teardown(tmpDir);
+    process.chdir(root);
   });
 
   function confirmBlueprintedForDir(blueprintDir, expectedAppDir = 'foo', typescript = false) {
-    let blueprintPath = path.join(root, blueprintDir, 'files');
+    let blueprintPath = path.join(blueprintDir, 'files');
```

Comment:
All the below assertions appear to be dynamic, so changing the test makes me nervous. Could you add an assertion below that expected isn't an empty array?

Reply:
sure 👍